doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -591,65 +620,20 @@ Induction
 
       This error is given when one side of the equality is not a constructor.
 
-   .. tacv:: injection @natural
-
-      This does the same thing as :n:`intros until @natural` followed by
-      :n:`injection @ident` where :n:`@ident` is the identifier for the last
-      introduced hypothesis.
-
-   .. tacv:: injection @term with @bindings
+   .. tacn:: einjection {? @induction_arg } {? as {* @simple_intropattern } }
 
-      This does the same as :n:`injection @term` but using the given bindings to
-      instantiate parameters or hypotheses of :n:`@term`.
-
-   .. tacv:: einjection @natural
-             einjection @term {? with @bindings}
-      :name: einjection; _
-
-      This works the same as :n:`injection` but if the type of :n:`@term`, or the
-      type of the hypothesis referred to by :n:`@natural`, has uninstantiated
+      Works the same as :n:`injection` but if the type of :n:`@one_term`, or the
+      type of the hypothesis referred to by :n:`@natural` has uninstantiated
       parameters, these parameters are left as existential variables.
 
-   .. tacv:: injection
-
-      If the current goal is of the form :n:`@term <> @term` , this behaves as
-      :n:`intro @ident; injection @ident`.
-
       .. exn:: goal does not satisfy the expected preconditions.
          :undocumented:
```

Comment:
It looks like this error message doesn't exist. I can't see what it would correspond to, so I propose to just remove it.